app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,99 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.models.TaskDto;
+import com.codesoom.assignment.services.TaskService;
+import org.modelmapper.ModelMapper;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import java.util.List;
+
+/**
+ * 할 일 관련 HTTP 요청을 처리하기 위한 클래스.
+ *
+ * @see TaskService
+ */
+@RestController
+@RequestMapping("/tasks")
+@CrossOrigin
+public class TaskController {
+
+    private final TaskService taskService;
+    private final ModelMapper modelMapper;
+
+    public TaskController(TaskService taskService, ModelMapper modelMapper) {
+        this.taskService = taskService;
+        this.modelMapper = modelMapper;
+    }
+
+    /**
+     * 모든 할일을 리턴합니다.
+     * @return 모든 할 일
+     */
+    @GetMapping
+    public List<Task> list() {
+        return taskService.getTasks();
+    }
+
+
+    /**
+     * 주어진 할 일을 저장한 뒤, 저장된 할 일, 201코드를 리턴합니다.
```

Comment:
`CREATED`가 애노테이션에 들어있는 정보라는 사실을 잊으면 안 됩니다.

```suggestion
     * 주어진 할 일을 저장한 뒤, 저장된 할 일을 리턴합니다.
```